### PR TITLE
add Project.toml

### PR DESCRIPTION
**Pull request by Alexander-Barth**
All test pass now locally using BinDeps 1.0 when a Project.toml file is added (#2)

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name = "CondaBinDeps"
+uuid = "a9693cdc-2bc8-5703-a9cd-1da358117377"
+version = "0.2.0"
+
+[deps]
+BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
```
Comment (stevengj) [inline Project.toml]: Do we still need `Compat` since we are now requiring Julia 1.x?

Comment (Alexander-Barth) [reply Project.toml]: You are right, Compat can indeed be dropped. I adapted the PR.

**Comment by Alexander-Barth** [comment]
If this PR is acceptable, can a new release be made? I would need this change so that my package remains installable even if BinDeps 1.0 is installed.

https://github.com/Alexander-Barth/NCDatasets.jl/issues/45

**Comment by Alexander-Barth** [comment]
Thanks a lot!

**Pull request merged**